### ISSUE TITLE
Image format errors

### ISSUE THREAD

**Issue by delphinepagesbv** [CalderaWP/Caldera-Forms#1914]
Hi, 
Typically when a user doesn't provide a required information or a valid value and tries to submit the form it would scroll back up to where the error occurred.  That is not true for the image format errors though. 
If a user uploads a PNG instead of a JPG for example, then scrolls down and clicks Submit nothing seems to happen. When I scroll back up I can see the message "This file type is not allowed. Please try another." but in this scenario it's not clear for a user if the form has been submitted or not (after clicking the submit button) and if there's an error.
It also seems to validate the format of the uploaded image only once all the other required fields have been filled in.

Debug info: WordPress Version: 4.8.1PHP Version: 5.6.31-4+wpengine10MySQL Version: 5.6.37Caldera Forms Version: 1.5.5WP_DEBUG:

Thanks,

Delphine

**Comment by Shelob9** [comment]
Thanks for using Caldera Forms. We can not accept bug reports that do not follow our template. I hope you understand that the information that we ask for would explain if this is a bug or a site-specific issue. Right now I have no idea what is going wrong on your site.

Please open a new ticket following the new issue template.

Please note, this is not a support channel.  If you need immediate help with your site or help troubleshooting Caldera Forms on your site, please open a support ticket at https://CalderaForms.com/support
